spec/index.md

```diff
@@ -427,6 +427,16 @@ The state tree contains information about the topology of the Internet Computer.
         principal = bytes .size (0..29)
         tagged<t> = #6.55799(t) ; the CBOR tag
 
+-   `/subnet/<subnet_id>/metrics` (blob)
+
+     A list of subnet wide metrics related to this subnet's current usage and/or performance. The metrics are a CBOR map with the following fields:
```

Comment:
```suggestion
     A collection of subnet-wide metrics related to this subnet's current resource usage and/or performance. The metrics are a CBOR map with the following fields:
```
I'm using "collection" instead of "list" as the blob is a CBOR "map".